src/dynamixel_main.cpp

```diff
@@ -0,0 +1,123 @@
+#include <fcntl.h>
+#include <termios.h>
+#define STDIN_FILENO 0
+
+#include <stdio.h>
+#include <chrono>
+#include <functional>
+#include <memory>
+#include <string>
+
+#include "dynamixel_sdk/dynamixel_sdk.h"
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/string.hpp"
+
+#define PROTOCOL_VERSION                2.0
+
+#define DXL_ID                          1
+#define BAUDRATE                        57600
+#define DEVICENAME                      "/dev/ttyUSB0"
+
+#define NODE_NAME                       "dynamixel_ping"
+#define TOPIC_NAME                      "dynamixel_topic"
+
+using namespace std;
+
+int getch()
```

Comment:
Remove the `getch()` and any kind of user input, make it as simple as possible. Run the node, shows the response, if failed, just exit.